arguments/songname.js

```diff
@@ -65,12 +69,18 @@ module.exports = class extends Argument {
     /**
      * Gets an array of tracks from lavalink REST API
      * @param {string} search The search string
+     * @param {Object} node The node to use for REST searches
      * @returns {Array<Object>}
      */
-    getTracks(search) {
-        return this.client.lavalink.getSongs(search);
+    getTracks(search, node) {
```

Comment:
node, arg.

this function should handle the new way of the Lavalink v3 REST API results. this just makes everything slower bc you can just use the resolveTracks function directly. but this function should definitely handle the new response of Lavalink v3